frb_codegen/src/main.rs

```diff
@@ -1,19 +1,27 @@
 use env_logger::Env;
+use lib_flutter_rust_bridge_codegen::{config_parse, ensure_tools_available, frb_codegen, RawOpts};
 use log::info;
 use structopt::StructOpt;
 
-use lib_flutter_rust_bridge_codegen::{frb_codegen, Opts};
-
-fn main() {
-    let opts = Opts::from_args();
-    env_logger::Builder::from_env(Env::default().default_filter_or(if opts.verbose {
+fn main() -> anyhow::Result<()> {
+    let raw_opts = RawOpts::from_args();
+    env_logger::Builder::from_env(Env::default().default_filter_or(if raw_opts.verbose {
         "debug"
     } else {
         "info"
     }))
     .init();
 
-    frb_codegen(opts).unwrap();
+    // initial config(s) before generation
+    ensure_tools_available()?;
```

Comment:
Maybe `ensure_tools_available` should be put in `lib`'s call, not `main`'s call, otherwise lib users will lack this sanity check?

Reply:
From the sake of lib users, I've put it back into `frb_codegen` in 9e38c2995f796d3e5c35b6bbaa631d62645325dc.